metaci/conftest.py

```diff
@@ -207,3 +212,12 @@ class Meta:
 
     branch = factory.SubFactory(BranchFactory)
     plan = factory.SubFactory(PlanFactory)
+
+
+class FlowTaskFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = FlowTask
+
+    build_flow = factory.SubFactory(BuildFlowFactory)
+    stepnum = factory.Sequence(lambda n: f"{n}.")
```

Comment:
Why'd you include the period here? Normal stepnums would like one of these:

    1  # a task that is the 1st step at the top level
    1.1  # a task that is the 2nd step at the top level (sorted numerically within a level)
    2/1  # task which is the first step within a flow that is the second step at the top level

so they can include periods, but it's odd to have one that's not being used as a separator

Reply:
hmmm. I don't recall why I did that. Might have been my ignorance about step numbers. I'll take a look.

Reply:
I couldn't find any reason for that change so I reverted it. You win the eagle-eye award for catching this!